agent/src/main/java/com/walmartlabs/concord/agent/cfg/DockerConfiguration.java

```diff
@@ -36,13 +36,15 @@
     private final boolean orphanSweeperEnabled;
     private final long orphanSweeperPeriod;
     private final List<String> extraVolumes;
+    private final Boolean exposeDockerDaemon;
```

Comment:
Lower-case `boolean` maybe? I don't think it's a real tri-state value?
The same in other files.

Reply:
Yep, I agree